stdlib/public/SDK/Network/NWConnection.swift

```diff
@@ -478,6 +478,7 @@ public final class NWConnection : CustomDebugStringConvertible {
 		}
 	}
 
+	/// Callbacks used when sending data to the protocol stack. 
```

Comment:
This should be singular. It’s a type that’s a lot like Optional, so the documentation should be as well. Something like: “A type that represents either a wrapped completion handler invoked when send content has been consumed by the protocol stack, or the lack of a completion handler because the content is idempotent.”